components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ExportUtils.java

```diff
@@ -778,10 +779,13 @@ public static void addAPIMetaInformationToArchive(String archivePath, APIDTO api
                                 exportFormat, ImportExportConstants.GRAPHQL_COMPLEXITY, graphQLQueryComplexityInfoDTO);
                     }
                 }
+                API api = APIMappingUtil.fromDTOtoAPI(apiDtoToReturn, apiDtoToReturn.getProvider());
+                //TODO-ORG
```

Comment:
What are the things remaining to do here?

Reply:
That has been already implemented. Removed that comment

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9684/commits/cc8517b20f7dbec93569cf9ab02809b6b9148cbe